### ISSUE TITLE
Update esmm dropdowns

### ISSUE THREAD

**Issue by daniel-tabarcea** [unicef/etools-issues#517]
Update dropdowns in the app to use etools-multi-selection-menu instead of etools-searchable-multiselection-menu

**Comment by daniel-tabarcea** [comment]
The updated Dropdowns are: 
 on PD/SSFA: 
  - list filters
  - CP Indicators
 - UNICEF Office(s)
 - UNICEF Focal Point(s)
 - Partner Focal Point(s)
 - Signed By Partner Authorized Officer (this one's a single select)
 - Signed by UNICEF (also a single) 
 - Section/Cluster (on sector locations) 
 - Locations (on sector locations)

on Partners: 
 - list filters
 - Shared Partner
 
on Agreements list: 
 - list filters 

Basically they need to work like they did before.

**Comment by roxanavlad** [comment]
Test result: pass

**Comment by mpawlowski85** [comment]
it looks to me like in staging, these filters are still multi select and searchable. 
However, I do not completely understand the problem. 
@daniel-tabarcea please close if this is no longer an issue in staging.
